### DISCUSSION TITLE
kill whole stack of mob function

### DISCUSSION THREAD

**Discussion by hkkongou** [OmerBenGera/WildStacker#260] [Denied Feature Requests] [answered]
A function for players to kill mobs instantly, so that mobs drop loot depending on the amount of stacked mobs.

this is different from one-shot or instant-kill, kill whole stack can save time for players so that if they have 100+ mobs don't have to click 100+ times, 1 killed, all 100+ mobs dies and 100+ item drops are given

**Comment by OmerBenGera** [comment, marked as answer]
sounds like instant-kill is the option you look for. Add "ENTITY_ATTACK" to instant-kill and it will work as you want.
